### DISCUSSION TITLE
21.2 Victoria is limping

### DISCUSSION THREAD

**Discussion by Papa-26** [linuxmint/discussions#133] [Questions]
I updated to Mint 21.2 via the update manager. I have had problems since with `sudo apt-get update`.
Investigating it, I found that the /etc/apt/source.lists still seem beholden to Vera!

**Comment by kacperpaczos** [comment]
I don't have any entries in this file, maybe it's just your configuration?

**Comment by anaximeno** [comment]
Followed up in https://github.com/linuxmint/mintupdate/issues/839
